EIPS/eip-6734.md

```diff
@@ -0,0 +1,574 @@
+
+<a name="r5"> **[R5]** </a> 
+A claim that a canonical token list implementation conforms to this specification at **Level 2** or higher MUST describe the testing procedure carried out for each requirement at **Level 2** or higher, that justifies the claim to that requirement.
+
+[[R5]](#r5) testability: Since each of the non-conformance-target requirements in this documents is testable, so must be the totality of the requirements in this document. Therefore, conformance tests for all requirements can exist, be described, be built and implemented and results can be recorded as required in [[R5]](#r5).
+
+-----
+
+## Rationale
+
+The standard is extending and clarifying current custom lists such as from Arbitrum and Optimism as referenced in the [Motivation](#motivation) or the Uniswap Tokenlist Project to improve clarity, security and encourage adoption by non-Web3 native entities.
+
+-----
+
+## Security Considerations
+
+There are no additional security requirements apart from the warnings that URIs utilized in implementations of this standard might be direct to malicious resources such as websites, and that implementers should ensure that data utilized for a canonical token list is secure and correct. Since this standard is focused on a data schema and its data properties there are no additional security considerations from for example homoglyph attacks (see [CVE-2021-42574](#cve-2021-42574)).
+
+### Security Considerations: Data Privacy
+
+The standard does not set any requirements for compliance to jurisdiction legislation/regulations. It is the responsibility of the implementer to comply with applicable data privacy laws.
+
+### Security Considerations: Production Readiness 
+
+The standard does not set any requirements for the use of specific applications/tools/libraries etc. The implementer should perform due diligence when selecting specific applications/tools/libraries.
+
+### Security Considerations: Internationalization and Localization
+
+The standard encourages implementers to follow the [W3C "Strings on the Web: Language and Direction Metadata" best practices guide](#w3c-string-meta) for identifying language and base direction for strings used on the Web wherever appropriate.
+
+-----
+
+### Appendix A - Normative References
+
+The following documents are referenced in such a way that some or all of their content constitute requirements of this document.
+
+<a name="rfc2119"> **[RFC2119]** </a>\
+ S. Bradner, Key words for use in RFCs to Indicate Requirement Levels, IETF RFC 2119, March 1997.
+
+<a name="jsonld"> **[JSONLD]** </a>\
+JSON-LD 1.1, M. Sporny, D. Longley, G. Kellogg, M. Lanthaler, Pierre-Antoine Champin, N. Lindström, W3C Recommendation, July 2020 
```

Comment:
W3C standards can be directly linked to using normal markdown syntax (using the format [mentioned in EIP-1](https://eips.ethereum.org/EIPS/eip-1#world-wide-web-consortium-w3c).)

```suggestion
```

Reply:
done